chain/arweave/Cargo.toml

```diff
@@ -22,3 +23,4 @@ graph-core = { path = "../../core" }
 graph-store-postgres = { path = "../../store/postgres" }
 pretty_assertions = "0.7.2"
 test-store = { path = "../../store/test-store" }
+hex = "0.4.3"
```

Comment:
I think you should probably use `graph::prelude::hex` instead of adding this dependency twice.

Reply:
well spotted, it's not actually needed anymore :) thanks